python/lsst/analysis/tools/tasks/catalogMatch.py

```diff
@@ -178,262 +112,175 @@ class CatalogMatchConfig(pipeBase.PipelineTaskConfig, pipelineConnections=Catalo
 
     extraColumns = pexConfig.ListField[str](
         doc="Other catalog columns to persist to downstream tasks",
-        default=["i_cModelFlux", "x", "y"],
+        default=["x", "y", "patch", "ebv"],
     )
 
-    requireProperMotion = pexConfig.Field[bool](
```

Comment:
Can you leave this in? I'm turning this on in a ticket I'm working on. I might be able to find a workaround if it's really annoying, since I see you aren't using the `meas_algorithms` reference loader directly anymore.